ui/admin/app/components/credential-stores/credential-store/actions/index.hbs

```diff
@@ -18,7 +18,7 @@
       {{t 'resources.credential.actions.create'}}
     </dd.Interactive>
   {{/if}}
-  {{#if (and (feature-flag 'vault-worker-filter') @model.isVault)}}
+  {{#if (and (feature-flag 'worker-filter') @model.isVault)}}
```

Comment:
Just wondering if we've verified this change in Launch Darkly?

Reply:
We only use launch darkly in HCP, so these our own feature flags

Reply:
As Zed mentioned, we use launchDarkly for feature flags found in https://github.com/hashicorp/cloud-ui/tree/main/engines/boundary

Reply:
Good to know thanks 👍